source/installation/deploy-minio-distributed.rst

```diff
@@ -498,4 +498,53 @@ host:
 
    * - More than 1 Pebibyte (Pi)
      - 128GiB
-       
+
+.. _minio-requests-per-node:
+
+Requests Per Node
+~~~~~~~~~~~~~~~~~
+
+You can calculate the maximum number of requests allowed per host with this formula:
```

Comment:
```suggestion
You can calculate the maximum number of concurrent requests per host with this formula:
```

We do want to specify max concurrent vs max requests, as the latter could be interpreted as the total number of connections over the lifetime of the node.

Reply:
👍